Test/Set-WhiskeyProGetAsset.Tests.ps1

```diff
@@ -0,0 +1,174 @@
+#Requires -Version 4
+Set-StrictMode -Version 'Latest'
+
+& (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-WhiskeyTest.ps1' -Resolve)
+
+$apiKey = 'HKgaAKWjjgB9YRrTbTpHzw=='
+$script:username = 'Admin'
+$credential = New-Credential -UserName $username -Password $username
+$script:taskParameter = @{ }
+$script:taskParameter['uri'] = ('http://{0}:82/' -f $env:COMPUTERNAME)

```

Comment:
Mock out all the calls to ProGet. Don't use a real ProGet server.

Reply:
Done